### PR TITLE
Allow changeing Congestion control algorithm at runtime

### PR DESCRIPTION
**Pull request by thiblahute**

### CODE REVIEW

**Comment by MathieuDuponchelle** [comment]
Why?

**Comment by thiblahute** [comment]
To demonstrate difference of behavior in real time, people like to see real results. Implementation is simple enough that it is worth it to me.

File: plugins/src/webrtcsink/imp.rs [hunk 1]
```diff
@@ -2084,9 +2084,29 @@ impl ObjectImpl for WebRTCSink {
...
+                let new_heuristic = value
                     .get::<WebRTCSinkCongestionControl>()
                     .expect("type checked upstream");
+                if new_heuristic != settings.cc_heuristic {
+                    settings.cc_heuristic = new_heuristic;
+
+                    let mut state = self.state.lock().unwrap();
+                    for (peer_id, consumer) in state.consumers.iter_mut() {
+                        match new_heuristic {
+                            WebRTCSinkCongestionControl::Disabled => {
```
Comment (MathieuDuponchelle) [inline plugins/src/webrtcsink/imp.rs:2096]: You want to set video encoder's bitrates to the max value in that case.

Comment (thiblahute) [reply plugins/src/webrtcsink/imp.rs:2099]: Ah, I forgot to cherry-pick https://github.com/centricular/webrtcsink/pull/8/commits/2d8f170261de768371933b30807fc539bd23f5d0

Comment (thiblahute) [reply plugins/src/webrtcsink/imp.rs:2099]: The behavior with that last patch is that we set the right bitrate right away.

Comment (MathieuDuponchelle) [reply plugins/src/webrtcsink/imp.rs:2099]: OK, but still you're not resetting the bitrate in the property

Comment (MathieuDuponchelle) [reply plugins/src/webrtcsink/imp.rs:2099]: merging for now, I'll fix locally

**Pull request merged**